eventhandling/configureEvent.go

```diff
@@ -370,7 +371,16 @@ func createScrapeJobConfig(scrapeConfig *prometheus.ScrapeConfig, config *promet
 	// define scrape job name
 	scrapeConfig.JobName = scrapeConfigName
 	// set scrape interval to 5 seconds
-	scrapeConfig.ScrapeInterval = prometheus_model.Duration(5 * time.Second)
+	scrapeInterval := utils.EnvVarOrDefault("SCRAPE_INTERVAL", "5")
+	scrapeIntervalInt, err := strconv.Atoi(scrapeInterval)
+
+	if err != nil {
+		logger.Error("Error while converting SCRAPE_INTERVAL value. Using default value instead!")
+		scrapeConfig.ScrapeInterval = prometheus_model.Duration(5 * time.Second)
+	} else {
+		scrapeConfig.ScrapeInterval = prometheus_model.Duration(time.Duration(scrapeIntervalInt) * time.Second)
+	}
+
```

Comment:
as per comment above, scrapeInterval should already be a `Duration` value coming from the caller